contribs/gnodev/pkg/dev/node_state.go

```diff
@@ -131,7 +131,6 @@ func (n *Node) ExportStateAsGenesis(ctx context.Context) (*bft.GenesisDoc, error
 
 	// Get current blockstore state
 	doc := *n.Node.GenesisDoc() // copy doc
-
 	genState := doc.AppState.(gnoland.GnoGenesisState)
 	genState.Balances = n.config.BalancesList
 	genState.Txs = state
```

Comment:
just a note that the deleted files below this file should be expanded for additional comments.